packages/next/build/webpack/plugins/serverless-plugin.ts

```diff
@@ -55,38 +55,18 @@ export class ServerlessPlugin {
   private buildId: string
   private isServer: boolean
   private isTrace: boolean
-  private isFlyingShuttle: boolean
 
   constructor(
     buildId: string,
-    {
-      isServer,
-      isTrace,
-      isFlyingShuttle,
-    }: { isServer: boolean; isTrace: boolean; isFlyingShuttle: boolean }
+    { isServer, isTrace }: { isServer: boolean; isTrace: boolean }
   ) {
     this.buildId = buildId
     this.isServer = isServer
     this.isTrace = isTrace
-    this.isFlyingShuttle = isFlyingShuttle
   }
 
   apply(compiler: Compiler) {
     if (!this.isServer) {
-      if (this.isFlyingShuttle) {
-        compiler.hooks.emit.tap('ServerlessPlugin', compilation => {
-          const assetNames = Object.keys(compilation.assets).filter(f =>
-            f.includes(this.buildId)
-          )
-          for (const name of assetNames) {
-            compilation.assets[
-              name
-                .replace(new RegExp(`${this.buildId}[\\/\\\\]`), 'client/')
-                .replace(/[.]js$/, `.${this.buildId}.js`)
-            ] = compilation.assets[name]
-          }
-        })
-      }
       return
```

Comment:
We don't need the `replaceInBuffer` logic anymore since we don't need to handle `dynamicBuildId` and can just handle it in `serverless-loader` again right?

Reply:
Build ID is still needed -- this is unrelated to flying shuttle. It's to skip re-emitting building a serverless page when nothing has changed IIRC. Note it's used for `buildId` still:
```js
        buildId: "__NEXT_REPLACE__BUILD_ID__",
```